package.json

```diff
@@ -4,29 +4,32 @@
   "scripts": {
     "test": "npm run lint && hardhat test",
     "fuzz": "hardhat compile && fuzzing/fuzz.sh",
-    "start": "hardhat node --export deployment-localhost.json",
+    "start": "concurrently --names \"hardhat,deployment\"  --prefix \"[{time} {name}]\" \"hardhat node\" \"sleep 2 && npm run mine && HARDHAT_NETWORK=localhost npm run deploy\"",
```

Comment:
wooow really?! Hardhat will not deploy the contracts automatically when starting the local network? That is weird. I mean, now it works like that... Well you managed to work around it which is great 👍